src/renderer/components/+entity-settings/internal-kubernetes-cluster/namespace-settings.injectable.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { withInjectables } from "@ogre-tools/injectable-react";
+import React from "react";
+import type { GetClusterById } from "../../../../common/cluster-store/get-by-id.injectable";
+import getClusterByIdInjectable from "../../../../common/cluster-store/get-by-id.injectable";
+import { ClusterAccessibleNamespaces } from "../../cluster-settings/accessible-namespaces";
+import type { EntitySettingViewProps } from "../extension-registrator.injectable";
+import { entitySettingInjectionToken } from "../token";
+
+interface Dependencies {
+  getClusterById: GetClusterById;
+}
+
+function NonInjectedProxyKubernetesClusterSettings({ entity, getClusterById }: EntitySettingViewProps & Dependencies) {
```

Comment:
Just to avoid confusion
```suggestion
function NonInjectedNamespaceKubernetesClusterSettings({ entity, getClusterById }: EntitySettingViewProps & Dependencies) {
```